test/quantization/core/test_quantized_op.py

```diff
@@ -4331,21 +4382,41 @@ def _test_qlinear_pt2e_helper(self, post_op_to_qlinear_ref_dict, supported_post_
                 )
 
     @skipIfNoONEDNN
-    def test_qlinear_linear_relu_pt2e(self):
-        post_op_to_qlinear_ref_dict = {
-            'none': torch.ops.quantized.linear,
-            'relu': torch.ops.quantized.linear_relu,
-        }
-        supported_post_ops = ['none', 'relu']
-        post_op_algorithms = []
-        self._test_qlinear_pt2e_helper(post_op_to_qlinear_ref_dict, supported_post_ops, post_op_algorithms)
+    def test_qlinear_pt2e(self):
```

Comment:
nit: as a follow up maybe you can add a new test_quantized_op_onednn.py file, so it's not confused with the existing quantized ops that is using quantized tensors.

Reply:
Makes sense. Thanks.